tests/test_cases/test_cocotb/test_timing_triggers.py

```diff
@@ -117,7 +117,8 @@ def do_test_afterdelay_in_readonly(dut, delay):
     exited = True
 
 
-@cocotb.test(expect_error=True,
+@cocotb.test(skip=cocotb.LANGUAGE in ["vhdl"] and cocotb.SIM_NAME.lower().startswith(("riviera")),
```

Comment:
Is Riviera also unexpectedly passing this test? Questa with the FLI does as well #1245. We probably need to enforce a particular behavior in the scheduler.

Reply:
```
Running test 43/91: test_readwrite_in_readonly
2020-06-14T08:15:17.0087217Z # COUT:   1120.43ns INFO     ..dwrite_in_readonly.0x7fde7438cb38       decorators.py:256  in _advance                        Starting test: "test_readwrite_in_readonly"
2020-06-14T08:15:17.0089812Z # COUT:                                                                                                                      Description: Test doing invalid sim operation
2020-06-14T08:15:17.0219891Z # COUT:   1120.58ns ERROR    cocotb.regression                         regression.py:366  in _score_test                     Test passed but we expected an error: test_readwrite_in_readonly (result was TestSuccess)
```